docs/wiki/getting-started/hardware/vtx.md

```diff
@@ -0,0 +1,41 @@
+# VTX
+
+Video Transmitters are commonly referred to as VTX units. These are typically analogue or digital systems transmitting on the 5.8GHz band, although other frequency bands are sometimes used for specialist long-range craft.
+
+## Video Systems
+
+Analogue video systems transmit a CVBS signal in either PAL or NTSC using channels in the 5.8GHz band.
```

Comment:
It might be good to make this into 
## Analog FPV video
and then make a separate subheading
$$ HD FPV video
where we discuss the MSP connection method, and configuration requirements for 03 and HDZero?